worker/main.go

```diff
@@ -20,7 +25,7 @@ func main() {
 	// This worker hosts both Worker and Activity functions
 	w := worker.New(c, "CART_TASK_QUEUE", worker.Options{})
 
-	var a *app.Activities
+	a := app.MakeActivities(stripeKey, mailgunDomain, mailgunKey)
```

Comment:
Out of curiousity why do you need this `Make` function? Why not just do a standard interface initialization

```go
a := &app.Activities{
  StripeKey: stripeKey,
  ...
}
```

Reply:
@rylandg I removed `MakeActivities()`, you're right that's unnecessary here. My reasoning was to keep `StripeKey` as a private field, but on second thought it isn't worth the extra mental overhead.